### PR TITLE
Feature/509 player UI refactor

### PR DESCRIPTION
**Pull request by lilitkarapetyan**
## 🔨Work carried out:

- [x] Tests pass

## Confirmations

- [ ] I have chosen reviewers for my PR.
- [ ] I have assigned myself to this PR.
- [ ] I have chosen an appropriate label for the PR.
- [ ] I have completed the mandatory sections of this document.
- [ ] I have deleted any unused sections.
- [ ] I confirm that I have checked for required README updates and acted accordingly.

### CODE REVIEW

**Comment by IanMayo** [comment]
This is a huge piece of work @lilitkarapetyan 

I guess there isn't a way of breaking it down into smaller PRs - you've just got to keep pushing it along.

File: packages/custom-types/asset.d.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Perception from './perception'
+
+export interface Asset {
+  name: string,
+  uniqid: string,
+  position: number,
+  perceptions: Perception[],
+  plannedTurns?: any[] // Todo Create type for plannedTurns
+}
+
```
Comment (IanMayo) [inline packages/custom-types/asset.d.ts:11]: Other fields: 
- platformType: string
- condition: string
- contactId: string
- status: status
- history: Step[]

See contents of `forces.mock.ts`

File: packages/custom-types/asset.d.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Perception from './perception'
+
+export interface Asset {
+  name: string,
+  uniqid: string,
+  position: number,
```
Comment (IanMayo) [inline packages/custom-types/asset.d.ts]: Position is a string:
image

File: packages/custom-types/asset.d.ts (hunk 1 above)
Comment (lilitkarapetyan) [reply packages/custom-types/asset.d.ts:11]: I think it's hostings
изображение

Comment (lilitkarapetyan) [reply packages/custom-types/asset.d.ts:11]: изображение

File: packages/custom-types/message.d.ts [hunk 3]
```diff
@@ -1,37 +1,45 @@
...
   open: boolean,
   isOpen: boolean
   hasBeenRead: boolean
   _id: string
-}
\ No newline at end of file
+  _rev?: string
+  gameTurn?: number,
+  feedback?: boolean,
+  infoType: undefined
```
Comment (IanMayo) [inline packages/custom-types/message.d.ts:44]: I _think_ `infoType` is an optional boolean.  Looking at the data, it's either `true` or missing.

**Review by IanMayo — APPROVED** [review]
I'm going to approve/merge this, so we can start using the new types.

**Pull request merged**